packages/headless/src/features/facets/facet-set/facet-set-slice.test.ts

```diff
@@ -299,6 +300,157 @@ describe('facet-set slice', () => {
     expect(() => facetSetReducer(state, action)).not.toThrow();
   });
 
+  describe('dispatching #toggleExcludeFacetValue with a registered facet id', () => {
+    const id = '1';
+    describe('when the facet value exists', () => {
+      it('sets the state of an idle value to excluded', () => {
+        const facetValue = buildMockFacetValue({value: 'TED'});
+        const facetValueRequest = convertFacetValueToRequest(facetValue);
+
+        state[id] = buildMockFacetSlice({
+          request: buildMockFacetRequest({currentValues: [facetValueRequest]}),
+        });
+
+        const action = toggleSelectFacetValue({
+          facetId: id,
+          selection: facetValue,
+        });
+        const finalState = facetSetReducer(state, action);
+
+        const targetValue = finalState[id]?.request.currentValues.find(
+          (req) => req.value === facetValue.value
+        );
+        expect(targetValue?.state).toBe('excluded');
+      });
+
+      it('sets the state of an excluded value to idle', () => {
```

Comment:
We will need tests for both `facet is selected -> execute toggleExclude` and `facet is excluded -> execute toggleSelect  (in the tests suite to toggle select).

From what I see it's already handled in facet-set-slice.